### ISSUE TITLE
Make "Body" part of the question optional?

### ISSUE THREAD

**Issue by fenbox** [apache/answer#842]
via [questions/D1xK2](https://meta.answer.dev/questions/D1xK2/make-body-part-of-the-question-optional)

## Is your enhancement request related to a problem? Please describe

> I run a community Q&A site where the questions are often very simple such as "Where do I get my dog groomed?". They fit into the Title of the question and there is no need to elaborate on it further in the body. However because the Body is a required field, people are often struggling as to what to put there.

## Describe the solution you'd like

> Can we introduce a feature where the Body field is optional and people can ignore it if they feel the Title captures the question sufficiently?

**Comment by surapuramakhil** [comment]
@fenbox check this. A hack / workaround for this. Better than suggested solution.

![image](https://github.com/apache/incubator-answer/assets/9161543/00608061-6bb4-4df9-a53e-170347e213fc)

**Comment by Octobug** [comment]
> Check this. A hack / workaround for this. This has a design advantage. Works same as how feed works in any social media apps

@surapuramakhil This workaround misses question level features such as bookmark and gamification (losing its nature).

@fenbox Should it be made configurable for admins? Like

#### Allow "Short questions"?

Description: Question content is not a required field for a short question.

---

And I'd like to fix this issue after the solution is settled.

**Comment by surapuramakhil** [comment]
@Octobug I agree what you mean. But users can do bookmark of a topic as such. As i this hack would work great if there would only up to 50 questions.

@fenbox @Octobug just my suggestion is if Q&A are too small then overhead of switching will be too much (1 page for each question). Having something like feed would be quite useful.

**Comment by fenbox** [comment]
@Octobug I prefer the second one.

Add "Min question body length" in Settings->Write. Default value is `6`, and `0` means optional.

![image](https://github.com/user-attachments/assets/0a6f13b4-2f61-4185-9b28-a7e6bfed9b2a)

**Comment by Octobug** [comment]
> [@Octobug](https://github.com/Octobug) I prefer the second one.
> 
> Add "Min question body length" in Settings->Write. Default value is `6`, and `0` means optional.
> 
> ![image](https://github.com/user-attachments/assets/0a6f13b4-2f61-4185-9b28-a7e6bfed9b2a)

@fenbox Sorry for not updating this for so long. I've been too busy last year... 🙇🏻 

Glad to see that salimina is working on it.

**Comment by fenbox** [comment]
@Dinesht04 Looking forward to it.
